release.json

```diff
@@ -3,15 +3,29 @@
         "INTEGRATIONS_CORE_VERSION": "master", 
         "OMNIBUS_SOFTWARE_VERSION": "master", 
         "OMNIBUS_RUBY_VERSION": "datadog-5.5.0", 
-        "JMXFETCH_VERSION": "0.34.0", 
-        "JMXFETCH_HASH": "b5c053b960f9fa949a76ec3b84c308586c409f956379481e6a0044867deb7b52"
+        "JMXFETCH_VERSION": "0.35.0", 
+        "JMXFETCH_HASH": "1f1c8435b56050e006990c450b3c8969c3097932c01e080b2c320343a968fcba"
     }, 
     "nightly-a7": {
         "INTEGRATIONS_CORE_VERSION": "master", 
         "OMNIBUS_SOFTWARE_VERSION": "master", 
         "OMNIBUS_RUBY_VERSION": "datadog-5.5.0", 
-        "JMXFETCH_VERSION": "0.34.0", 
-        "JMXFETCH_HASH": "b5c053b960f9fa949a76ec3b84c308586c409f956379481e6a0044867deb7b52"
+        "JMXFETCH_VERSION": "0.35.0", 
+        "JMXFETCH_HASH": "1f1c8435b56050e006990c450b3c8969c3097932c01e080b2c320343a968fcba"
+    }, 
+    "6.18.0-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.18.0-rc.1", 
+        "OMNIBUS_SOFTWARE_VERSION": "6.18.0-rc.1", 
```

Comment:
Are we going to create two different tags in `omnibus-software` now? Last release we only created the 7 tag (because there's no real point to having two different tags as long as 6 and 7 do the same thing).

Reply:
Both tags are present for 7.17.0-rc.1 but the 6 one was probably not used? I don't mind using only 7.18.0-rc.1, I'll change that.

Reply:
@julien-lebot that's nice I didn't know about it, it should probably be documented in the release doc
@KSerrania updated the PR